plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -215,6 +216,13 @@ public void storeMergedRows(Page page)
             for (int position = 0; position < page.getPositionCount(); position++) {
                 long operation = TINYINT.getLong(operationBlock, position);
 
+                checkState(operation <= UPDATE_OPERATION_NUMBER,
+                        format("Invalid operation value, supported are " +
```

Comment:
`checkState` takes a message format, to the purpose of lazy formatting of the exception message (usually check passes and so the message is not needed)

remove `format(`